src/createAuthUser.js

```diff
@@ -100,20 +102,33 @@ const createAuthUser = ({
 
   let tokenString = null // used for serialization
   if (firebaseUserClientSDK) {
+    /**
+     * Claims are injected client side through the onTokenChange Callback
+     */
+    claims = firebaseUserClientSDK.claims
     userId = firebaseUserClientSDK.uid
     email = firebaseUserClientSDK.email
     emailVerified = firebaseUserClientSDK.emailVerified
     getIdTokenFunc = async () => firebaseUserClientSDK.getIdToken()
     signOut = async () => firebase.auth().signOut()
     tokenString = null
   } else if (firebaseUserAdminSDK) {
+    /**
+     * firebaseUserAdminSDK is a DecodedIDToken obtained from
+     * admin.auth().verifyIdToken which returns all the user's claims
+     * https://firebase.google.com/docs/auth/admin/custom-claims
+     * In order for the claims to be consistent, we need to pass the
+     * entire adminSDK object as claims
+     */
+    claims = { ...firebaseUserAdminSDK }
```

Comment:
I think we'd want to filter out all standard claims properties here so it includes only custom claims. This will match the client-side implementation and also reduce the size of AuthUser.